src/main/kotlin/solve/importer/view/ImporterView.kt

```diff
@@ -1,17 +1,30 @@
 package solve.importer.view
 
-import javafx.geometry.Insets
+import io.github.palexdev.materialfx.css.themes.MFXThemeManager
+import io.github.palexdev.materialfx.css.themes.Themes
+import javafx.stage.DirectoryChooser
 import javafx.stage.Stage
-import tornadofx.borderpane
+import solve.importer.controller.ImporterController
 import tornadofx.View
+import tornadofx.borderpane
+
+class ImporterView : View() {
+
+    val importer = find<DirectoryPathView>()
+
+    private val directoryChooser = DirectoryChooser().apply { title = "Choose working directory" }
+
+    private val controller: ImporterController by inject()
 
-class ImporterView : View("Choose working directory") {
     override fun onDock() {
         (root.scene.window as Stage).minWidth = 390.0
+        MFXThemeManager.addOn(root.scene, Themes.DEFAULT)
     }
 
     override val root = borderpane {
-        padding = Insets(8.0, 10.0, 8.0, 10.0)
+        style="-fx-background-color: #ffffff;"
```

Comment:
Invalid style `WARNING: Caught 'java.lang.ClassCastException: class java.lang.Double cannot be cast to class javafx.scene.paint.Paint (java.lang.Double is in module java.base of loader 'bootstrap'; javafx.scene.paint.Paint is in unnamed module of loader 'app')' while converting value for '-fx-background-color' from inline style on VBox@19808b4`